app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -715,6 +716,14 @@ private void clearOverlay() {
                 | AppBarLayout.LayoutParams.SCROLL_FLAG_ENTER_ALWAYS);
     }
 
+    //To show the appbar
+    private void showAppBar() {
+        if (toolbar.getParent() instanceof AppBarLayout) {
+            ((AppBarLayout)toolbar.getParent()).setExpanded(true, true);
+        }
+
```

Comment:
Please remove this extra space and also i think there is no need of the comment as the method name is very much self-explanatory